docs/book/v3/getting-started/quick-start.md

```diff
@@ -590,15 +590,16 @@ The above looks for a query string parameter "target", and uses its value to
 provide to the template, which is then rendered and returned in an HTML
 response.
 
-Now, let's edit the template file to have the following contents:
+Now, let's edit the template file to have the one of the following header lines (use the one for your chosen template renderer):
 
 ```php
-<h1>Hello <?= $this->target ?></h1>
-```
+// plates
```

Comment:
Sorry, a misunderstanding. The code example is HTML _and_ PHP (PHTML), but the relevant part is PHP. Therefore we use "```php" in the documentation for this kind of code examples.
It would be better if we can use "```php+html", but I don't know if this work.

So please revert to HTML comment syntax.

Reply:
Reverted back to html.

Reply:
@xtreamwayz 
I found this:

> There has to be at least one <?php ?> tag in the source code, for Prism to detect if it is a HTML page with PHP tags included or simple plain PHP code.

https://github.com/PrismJS/prism/issues/1238#issuecomment-348593975